### ISSUE TITLE
Base commands should be overridable

### ISSUE THREAD

**Issue by kyouko-taiga** [zackargyle/react-bash#23]
It is currently impossible to override the base commands, although it might be desirable for a user to change the behaviour of a given command. One possible use case might be to link a structure modifying command to an actual request to an external API:

```js
import * as builtin from 'react-bash/commands';

const mkdir = {
    exec: (state, { args }) => {
        someExternalAPI.execute('mkdir', args);
        return builtin.mkdir(state, args);
    },
};

const extensions = { mkdir };
<Terminal extensions={extensions} />
```

I think it should suffice to replace [src/bash.js:9](https://github.com/zackargyle/react-bash/blob/master/src/bash.js#L9) with the following:

```js
this.commands = Object.assign({}, BaseCommands, extensions);
```

**Comment by irth** [comment]
I implemented the proposed change.

**Comment by zackargyle** [comment]
This was fixed in #24

**Comment by irth** [comment]
Cool, thanks!
